### DISCUSSION TITLE
How can I build a docker image from dockerfile with REST API?

### DISCUSSION THREAD

**Discussion by dayeguilaiye** [quay/quay#1869] [Q&A]
Is there a REST API, that I can upload Dockerfile and other files, then build the image on QUAY? I didn't find it.

Thanks for any help!

**Comment by codespearhead** [comment]
To my understanding, that's out of scope for the Quay project unless you use Quay Operators, which is probably an overkill for your use case.

If I were you, I would use [build triggers](https://docs.quay.io/guides/building.html) with a remote git repository, such as GitLab.
